src/Drivers/nlpSparse_ex7.cpp

```diff
@@ -7,14 +7,14 @@
 /** Nonlinear *highly nonconvex* and *rank deficient* problem test for the Filter IPM
  * Newton of HiOp. It uses a Sparse NLP formulation. The problem is based on Ex6.
  *
- *  min   -(2*convex_obj-1)*sum 1/4* { (x_{i}-1)^4 : i=1,...,n} + 0.5x^Tx
+ *  min   (2*convex_obj-1)*scal*sum 1/4* { (x_{i}-1)^4 : i=1,...,n} + 0.5x^Tx
```

Comment:
please document `scal`. Also not sure the variable name `scal_neg_obj` used later on is the best one. Looks like a better name would be `scale_quartic_obj_term` ...